packages/interface/src/utils/fetchTally.ts

```diff
@@ -48,7 +48,9 @@ export async function fetchTally(id: string): Promise<Tally | undefined> {
     body: JSON.stringify({
       query: tallyQuery.replace("id: $id", `id: "${id}"`),
     }),
-  }).then((response: GraphQLResponse) => response.data?.tally);
+  })
+    .then((response: GraphQLResponse) => response.data?.tally)
+    .catch(() => undefined);
```

Comment:
should this also return [] rather than undefined?

Reply:
this is just `fetchTally`, and this is actually not used, what you mentioned should be `fetchTallies`.
for `fetchTally` it accept `undefined` or `Tally`.